### PR TITLE
recipe/CEN-6951_video_to_image

### PR DESCRIPTION
**Pull request by strikeone88**
Completed recipe.

### CODE REVIEW

**Review by fayyazul-centaurlabs — APPROVED** [review]
Great work! Nice bonus portion as well.

After you get this merged in please also make the google document so we can get this on the docs website.

File: video_to_image/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+numpy==1.26.0
+opencv-python==4.8.0.76
```
Comment (gordonwade) [inline video_to_image/requirements.txt:2]: In order for the [top-level instructions](https://github.com/centaurlabs/centaur-recipes#installing-requirements-and-running-a-notebook) to work, we need `jupyter` and `ipykernel` installed as well.
```suggestion
jupyter
ipykernel
numpy==1.26.0
opencv-python==4.8.0.76

```

File: video_to_image/main.ipynb [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    "# Video portion to extract (in seconds).\n",
+    "START_TIME = 1.5\n",
+    "END_TIME = 1.8\n",
+    "\n",
+    "# Set the appropriate initial frame and calculate frames to extract.\n",
+    "video.set(cv2.CAP_PROP_POS_FRAMES, START_TIME * fps)\n",
+    "total = int((END_TIME - START_TIME) * fps)\n",
+    "\n",
+    "index = 0\n",
+    "while index < total:\n",
```
Comment (gordonwade) [inline video_to_image/main.ipynb:94]: Should this be `<` or `<=`? The instructions should make it clear whether the interval is inclusive.

**Review by gordonwade — COMMENTED** [review]
Hey @strikeone88 , this looks great! Just left one minor change and one question.

Because these are public-facing recipes, it is important to make sure that everything works properly when following the instructions as written. Following the [top-level instructions](https://github.com/centaurlabs/centaur-recipes#installing-requirements-and-running-a-notebook) also requires installation of `jupyter` and `ipykernel` which are not specified in the `requirements.txt` file.

Basically, we want to make sure to do a trial run on every recipe before publishing to make sure they are seamless!

**Comment by ciachn** [comment]
@gordonwade I have updated with the necessary changes. Please review.

**Review by gordonwade — APPROVED** [review]
Thanks, @strikeone88 !
